mmcv/runner/hooks/optimizer.py

```diff
@@ -40,11 +40,22 @@ class GradientCumulativeOptimizerHook(OptimizerHook):
     def __init__(self, grad_clip=None, cumulative_iters=1):
         super(GradientCumulativeOptimizerHook, self).__init__(grad_clip)
         self.cumulative_iters = cumulative_iters
+        self.divisible_ietrs = 0
+        self.remainder_iters = 0
+        self.initialized = False
+
+    def _init(self, runner):
+        self.divisible_ietrs = runner.max_iters // self.cumulative_iters * self.cumulative_iters
```

Comment:
There is another corner case where users resume from `iter=2` but the `cumulative_iters=4`. It seems this implementation will bring wrong gradients. If no good solutions, please put a warning here.